pkgs/by-name/sh/shh/package.nix

```
@@ -0,0 +1,34 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, python3
+, strace
+, systemd
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "shh";
+  version = "2023.12.16";
```

Latest release is now [v2024.4.5](https://github.com/desbma/shh/releases/tag/v2024.4.5)

```suggestion
  version = "2024.4.5";
```